fixtures/receivers/README.md

```diff
@@ -0,0 +1,44 @@
+# NFL Receiver Statistics from the 2018 Regular Season
```

Comment:
Thank you so much for this excellent README! I appreciate that you wrote it in the style of the other READMEs - it definitely makes it easier to digest for our users who are dealing with the data!

Reply:
Great job with the README @mike-curry00! Having a consistent style is especially helpful for new users, so I really appreciate that you took the time to review our documentation and write the README in a similar style.